lib/hashes.md

```diff
@@ -5,40 +5,46 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A hash is defined by curly brackets {} and can be viewed as a form of an array
+except the order or list of values are not important with hashes as it is with arrays.
```

Comment:
Exactly! Rather than thinking of a Hash as kind of like an Array, I find it helpful to think Arrays and Hashes as different kinds of collections.